scripts/install/install_unix.sh

```diff
@@ -726,14 +739,14 @@ display_results()
     increase_indent
     info "Agent Home:         $(fg_cyan "/opt/observiq-otel-collector")$(reset)"
     info "Agent Config:       $(fg_cyan "/opt/observiq-otel-collector/config.yaml")$(reset)"
+    info "Agent Logs Command:       $(fg_cyan "sudo tail -F /opt/observiq-otel-collector/agent.log")$(reset)"
```

Comment:
This is still the agent file, but changed to this because the supervisor doesn't allow configuring agent logging quite yet, so for the time being it directs agent logs to a file `agent.log` where the supervisor is running, so updated this command to match that